runtime/modules/java/core/src/main/java/io/atlasmap/java/inspect/ClassInspectionService.java

```diff
@@ -55,143 +52,71 @@
     public static final int MAX_ARRAY_DIM_LIMIT = 256; // JVM specification
 
     private static final Logger LOG = LoggerFactory.getLogger(ClassInspectionService.class);
-                                                       // limit
+    // limit
 
     private List<String> listClasses = new ArrayList<String>(
             Arrays.asList("java.util.List", "java.util.ArrayList", "java.util.LinkedList", "java.util.Vector",
                     "java.util.Stack", "java.util.AbstractList", "java.util.AbstractSequentialList"));
 
     private AtlasConversionService atlasConversionService = null;
     private List<String> fieldBlacklist = new ArrayList<String>(Arrays.asList("serialVersionUID"));
-    private List<String> classNameBlacklist = new ArrayList<String>();
     private Boolean disablePackagePrivateOnlyFields = false;
     private Boolean disableProtectedOnlyFields = false;
     private Boolean disablePrivateOnlyFields = false;
     private Boolean disablePublicOnlyFields = false;
     private Boolean disablePublicGetterSetterFields = false;
 
-    public List<String> getListClasses() {
+    private List<String> getListClasses() {
         return this.listClasses;
     }
 
-    public List<String> getClassNameBlacklist() {
```

Comment:
I don't think we have to remove getters even if it's not used today...

Reply:
Class variable classNameBlacklist was removed as it has no references. So getClassNameBlacklist also removed.

Reply:
Class variable classNameBlacklist was removed as it has no references,so corresponding getxxx method also removed